### PR TITLE
Fix UnicodeDecodeError problem

### PR DESCRIPTION
**Pull request by kawacchu**
Close: #602 

I tested locally.
I need your help if we make test function :bird:

### CODE REVIEW

**Comment by fukatani** [comment]
In Linux, How about using `echo $'\xff'` as command to implement test?

**Comment by kawacchu** [comment]
Due to lack of my knowledge of file operations and codecs, the test implementation is being delayed, I'm sorry!

**Comment by fukatani** [comment]
OK.
If implementing test is difficult, you can split it to another issue.

**Comment by kmyk** [comment]
I have implemented the test for this pull request.
@kawacchu please `cherry-pick` https://github.com/kmyk/online-judge-tools/commit/d8e0301c30e12abb82ee72708833a6750b4920a8

File: tests/command_test.py [hunk 1]
```diff
@@ -903,3 +903,23 @@ def test_call_stderr_and_fail(self):
                 'exitcode': 255,
             }],
         )
+
+    def test_call_non_unicode(self):
+        data = self.snippet_call_test(
+            args=['-c', tests.utils.python_c("import sys; sys.stdout.buffer.write(bytes(range(256)))")],
```
Comment (kawacchu) [inline tests/command_test.py]: 🙇🏻 🙇🏼 🙇🏽 🙇🏾 🙇🏿

**Pull request merged**